core/src/main/java/org/opensearch/sql/expression/operator/arthmetic/MathematicalFunction.java

```diff
@@ -197,6 +198,18 @@ private static DefaultFunctionResolver exp() {
                 type, DOUBLE)).collect(Collectors.toList()));
   }
 
+  /**
+   * Definition of expm1(x) function. Calculate exponent function e to the x, minus 1
+   * The supported signature of exp function is INTEGER/LONG/FLOAT/DOUBLE -> DOUBLE
+   */
+  private static DefaultFunctionResolver expm1() {
+    return FunctionDSL.define(BuiltinFunctionName.EXPM1.getName(),
+        ExprCoreType.numberTypes().stream()
+            .map(type -> FunctionDSL.impl(FunctionDSL.nullMissingHandling(
+                v -> new ExprDoubleValue(Math.expm1(v.doubleValue()))),
+                type, DOUBLE)).collect(Collectors.toList()));
+  }
```

Comment:
Is it possible to extract common code piece to another function? It could be re-used across all other Math functions.

Reply:
+1